app/view_models/navigation.rb

```diff
@@ -0,0 +1,41 @@
+class Navigation
+  class << self
+    include Rails.application.routes.url_helpers
+  end
+  NavItem = Struct.new(:title, :href, :children)
+
+  def self.navigation_items(user)
```

Comment:
I think the fact that this takes a `user` means it doesn't necessarily need to be a class method? And then we could get rid of the metaclass include and just do a regular include?

WDYT of:

```ruby
Navigation.new(user).navigation_items
```

Reply:
I think it makes a lot of sense, switched it to do that